ramls/item.json

```diff
@@ -200,6 +200,11 @@
     "temporaryLocationId": {
       "type": "string"
     },
+    "effectiveLocationId": {
```

Comment:
Given this is expected to be a UUID, even though it is read only, it could be useful to clients to be explicit about this by including the UUID pattern for this property.

Reply:
@marcjohnson-kint, thanks, fixed.